flotilla/endpoints.go

```diff
@@ -344,7 +356,21 @@ func (ep *endpoints) CreateRunV2(w http.ResponseWriter, r *http.Request) {
 	} else {
 		lr.Engine = &state.DefaultEngine
 	}
-	run, err := ep.executionService.Create(vars["definition_id"], lr.ClusterName, lr.Env, lr.RunTags.OwnerEmail, nil, nil, nil, lr.Engine, nil, nil)
+
+	req := state.DefinitionExecutionRequest{
+		ExecutionRequestCommon: state.ExecutionRequestCommon{
```

Comment:
Maybe an opportunity to make this more DRY, we have the same block repeated a few times.